syntax/NeogitStatus.vim

```diff
@@ -5,9 +5,9 @@ endif
 source $VIMRUNTIME/syntax/gitrebase.vim
 
 syn match NeogitObjectId /^[a-z0-9]\{7,}\>/
-syn match NeogitCommitMessage /.*/ contained
-syn match NeogitBranch /\S\+/ contained nextgroup=NeogitCommitMessage
-syn match NeogitRemote /\S\+/ contained nextgroup=NeogitCommitMessage
+syn match gitcommit /.*/ contained
+syn match NeogitBranch /\S\+/ contained nextgroup=gitcommit
+syn match NeogitRemote /\S\+/ contained nextgroup=gitcommit
```

Comment:
I am not entirely sure but those probably can just be removed. Nvim comes with gitcommit syntax as well as there is treesitter parser for gitconmit now.

Reply:
I'll try removing

Reply:
Yea, these can be removed altogether.

Reply:
An update on this: These should _not_ be removed, and should be left unchanged. I was mistaken before: they have nothing to do with the NeogitGitCommit popup. These three lines are just for syntax highlighting on the status buffer of the branch name/remote name.